site/specs/voice.yml

```diff
@@ -0,0 +1,5173 @@
+openapi: 3.0.3
+info:
+  title: Voice
+  version: 4.1.0
+  contact:
+    name: Bandwidth
+    url: https://support.bandwidth.com
+    email: support@bandwidth.com
+  description: |-
+    The API Specification for Bandwidth's Programmable Voice Platform
+
+    ## Base URL
+
+    `https://voice.bandwidth.com`
+servers:
+  - url: https://voice.bandwidth.com/api/v2
+    description: Production
+paths:
+  /accounts/{accountId}/calls:
+    post:
+      tags:
+        - Calls
+      summary: Create Call
+      description: >-
```

Comment:
Typo: '>' -> '|'

Reply:
Huh - these are all `|-` in my api-specs PR

Suspecting this has to do with the logic that adds in the code to the spec - we convert YAML to JSON and back to YAML

Let me investigate this